react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export default class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      drawerClassNames: 'drawer hidden',
+      containerClass: 'container'
+    };
```

Comment:
Try reworking this to just store whether or not the app drawer is open. Usually storing css classes in your state means you are storing the wrong thing in state.